script/clean.py

```diff
@@ -0,0 +1,61 @@
+import csv
+from os import sep
+import unidecode
+import pandas as pd
+
+path = ''
+file = open(path + "amajambo.csv")
+csvreader = csv.reader(file)
+
+rows = []
+for row in csvreader:
+    rows.append(row)
+
+flat_list = []
+for sublist in rows:
+    for item in sublist:
+        flat_list.append(item)
+
+words_list = [item.upper() for item in flat_list]
+
+bad_chars = ['!', "-"]
+
+for i in range(len(words_list)):
+    # remove ascents
+    words_list[i] = unidecode.unidecode(words_list[i])
+    words_list[i] = ''.join((filter(lambda i: i not in bad_chars, words_list[i])))
+
+#save all clean words in csv file
+with open('amajambo_cleanedV1.csv', 'w', newline='') as f:
+  csv_out = csv.writer(f)
+  csv_out.writerows([words_list[index]] for index in range(0, len(words_list)))
+
+#count number of words in our list
+def get_number_of_elements(list):
+    count = 0
+    for element in list:
+        count += 1
+    return count
```

Comment:
I think you can even simplify this method like this? What do you think ?

```suggestion
def  (list):
    len(list)
```

Maybe then replace `get_number_of_elements` with just `len`

Reply:
I'll keep this as it is at the moment. it's not working.

Reply:
What is the error you have ? I would be surprised that we need to do so much gymnastic to compute the size of a list in Python

Reply:
returns nothing if I try your approach